fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -141,6 +146,10 @@
 
     protected abstract String externalPath();
 
+    protected Deskolemizer deskolemizer() {
```

Comment:
It does not appear that this method is used outside of this class. Is there a reason for not making it `private`?